project/Common.scala

```diff
@@ -41,7 +41,7 @@ object Common {
     dockerCmd := Seq(),
     dockerCommands := dockerCommands.value.flatMap {
       case cmd @ Cmd("WORKDIR", _) =>
-        Seq(Cmd("RUN", "apk add --no-cache --update bash docker"), cmd)
+        Seq(Cmd("ENV", s"APP_VERSION=${version.value}"), Cmd("RUN", "apk add --no-cache --update bash docker"), cmd)
```

Comment:
Seems a bit off to me to be a an env variable, and not something that it is like part of the jar itself. But maybe is fine, would need to check better to come with an alternative

Reply:
🙏  yeah this was my first approach, I can take a look at using something that does not rely on env later today or in a bit of my offline day tomorrow

Reply:
We already have some versions available in code for some tools, e.g., [checkstyle](https://github.com/codacy/codacy-checkstyle/blob/0a7042a3894e4183b3b81d6dbd170276747336ac/build.sbt#L14).

Maybe there are better ways of doing it though 🤔

Reply:
Maybe there are, but I do prefer that one that you mentioned @franciscodua as at least is something that then comes from the CLI itself and not the env where it is running